### DISCUSSION TITLE
Auto play on mobile device

### DISCUSSION THREAD

**Discussion by yashtandon113** [ant-media/Ant-Media-Server#5374] [FAQs]
I'm using AMS v2.6.2 and streaming WebRTC, but the stream doesn't auto play on mobile devices but does on PC browsers. Is there a way to solve this problem?

**Comment by yashtandon113** [comment]
In v2.6 and above, you need to edit the `embedded-player.js` file under the `/usr/local/antmedia/webapps/App-Name/js` folder and set the below part to true.

```
if (isMobile()) {
            this.autoPlay = true;        
   }
```

After that, the stream will begin playing automatically on Android mobile devices, but you must consider the following steps for iOS.

1. The video tag should include the `playsinline` attribute. Check out the Apple developer forum [here](https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari#3030251).

To do so, update the same embedded-player.js file and alter the following parameters as shown below.

`EmbeddedPlayer.VIDEO_HTML = "<video id='video-player' class='video-js vjs-default-skin vjs-big-play-centered' controls playsinline></video>";`

2. Even after adding the playsinline property to the video element, some users may still experience the same issue, which is dependent on the user's device settings. iOS 10 and later allow video autoplay inline, however you must disable **"Low power mode"** on your iPhone.

**Reply by mekya** [reply]
You're right.  In 2.6, we've changed the player and we can still have some issues. 

I've sent a PR that may resolve this issue. Could you please try in your convenient time?
https://github.com/ant-media/StreamApp/pull/393

**Reply by yashtandon113** [reply]
The solution has been tested and is working. Thank you, @mekya, for resolving the issue permanently.

Members of the community, this permanent fix will be accessible beginning with v2.6.3.
